main.cpp

```diff
@@ -41,12 +41,207 @@
 #endif
 
 #ifdef CUSTOM_MUTATOR_BOLT11
-#include <modules/bolt11mutator/bech32.h>
+#include <modules/custommutator/bech32.h>
+#include <modules/custommutator/customcrossover.h>
 #endif
 
 
+#if defined(CUSTOM_MUTATOR_BOLT11) || defined(CUSTOM_MUTATOR_BOLT12_OFFER)
+#include <modules/custommutator/bech32.h>
+#include <modules/custommutator/customcrossover.h>
+#endif
+
+#ifdef CUSTOM_MUTATOR_BOLT12_OFFER
+extern const std::string bech32_hrp = "lno";
+extern const std::string dummy_initial_input = "lno1zcss9mk8y3wkklfvevcrszlmu23kfrxh49px20665dqwmn4p72pksese";
```

Comment:
I don't think there's any value in using `extern` here.  Also these can be compile-time constants.
```suggestion
constexpr std::string_view bech32_hrp = "lno";
constexpr std::string_view dummy_initial_input = "lno1zcss9mk8y3wkklfvevcrszlmu23kfrxh49px20665dqwmn4p72pksese";
```

Also let's just move these definitions to the below ifdef instead?

In fact `dummy_initial_input` could be defined inside `initial_input`.

Reply:
The idea of defining it externally is to reuse this CUSTOM_MUTATOR_BOLT12 for other BOLT12 targets, such as invoice_request and invoice.

Reply:
Sure, but we don't do that yet, and the ifdefs are currently the same (which needs to be changed later anyway), so let's just combine them for now?

Reply:
Okay